scripts/validate_string_concatenation.py

```diff
@@ -87,43 +251,37 @@ def main(source_path: str, output_format: str) -> bool:
     return is_failed
 
 
-def strings_to_concatenate(source_path: str) -> Generator[Tuple[str, int], None, None]:
-    """
-    Yielding the strings that needs to be concatenated in a given file.
-
-    Parameters
-    ----------
-    source_path : str
-        File path pointing to a single file.
-
-    Yields
-    ------
-    source_path : str
-        Source file path.
-    line_number : int
-        Line number of unconcatenated string.
-    """
-    with open(source_path, "r") as file_name:
-        tokens: List = list(tokenize.generate_tokens(file_name.readline))
-
-    for current_token, next_token in zip(tokens, tokens[1:]):
-        if current_token[0] == next_token[0] == token.STRING:
-            yield source_path, current_token[2][0]
-
-
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description="Validate concatenated strings")
+    available_validation_types = [
+        f.__name__
+        for f in globals().values()
+        if type(f) == type(main) and f.__name__ != "main"
+    ]
```

Comment:
I think it's probably better to have the list of functions explicit. So, it's clearer, and we have control on what can be called.